### PR TITLE
update to operator sdk

### PR DESCRIPTION
**Pull request by kgcarr**
Huge update, need to get added for main.go to compile

### CODE REVIEW

**Review by horis233 — COMMENTED** [review]
@kgcarr I have made some investigations about the operator-sdk upgrade base on the 3Q release mongodb operator.  You can check this dev branch in my repo https://github.com/horis233/ibm-mongodb-operator/tree/jiaming-dev

**Comment by kgcarr** [comment]
```
ibm-mongodb-operator-6fcc7d648d-g8sxz       1/1     Running   0          7m34s
icp-mongodb-0                               2/2     Running   0          7m13s
icp-mongodb-1                               2/2     Running   0          4m11s
icp-mongodb-2                               2/2     Running   0          2m2s
```

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```
Comment (horis233) [inline Dockerfile:2]: Please use golang:1.14.7 as the base image. 1.13 has been found some vulnerability issues.

**Pull request merged**